ostelco-core/Models/StageDecider.swift

```diff
@@ -11,22 +11,24 @@ import Foundation
 struct LocalContext {
     let selectedRegion: Region?
     let hasSeenLoginCarousel: Bool
-    let enteredEmailAddress: String?
-    let hasFirebaseToken: Bool
+    let enteredEmailAddress: String? // Needs to be persisted
+    let hasFirebaseToken: Bool // Needs to be persisted
     let hasAgreedToTerms: Bool
     let hasSeenNotificationPermissions: Bool
     let regionVerified: Bool
     let hasSeenVerifyIdentifyOnboarding: Bool
     let selectedVerificationOption: StageDecider.IdentityVerificationOption?
-    let myInfoCode: String?
+    let myInfoCode: String? // Needs to be persisted
     let hasSeenESimOnboarding: Bool
     let hasSeenESIMInstructions: Bool
     let hasSeenAwesome: Bool
-    let hasCompletedNRIC: Bool
-    let hasCompletedJumio: Bool
+    let hasCompletedJumio: Bool // Needs to be persisted
     let hasCompletedAddress: Bool
+    let serverIsUnreachable: Bool
+    let hasLocationProblem: Bool
```

Comment:
What kind of problem,  no permissions for accessing location?

Reply:
It doesn't matter specifically because currently the Location Problem view controller figured that out for itself.